book/src/qobject/rustobj_struct.md

```diff
@@ -20,12 +20,17 @@ The RustObj struct allows you to define the following items
 
 ## Invokables
 
-To define a method which is exposed to QML and C++, add a method on the `RustObj` struct and add the attribute `#[invokable]`. The parameters and return type are then matched to the Qt side. Also CXX-Qt automatically adds wrapper code around your invokable to automatically perform any conversion between the [C++ and Rust types](../concepts/types.md).
+A `impl cxx_qt::QObject<RustObj>` is used to define invokables, the `impl cxx_qt::QObject<RustObj>` defines that the methods are implemented onto the C++ QObject.
+Therefore they have access to both C++ and Rust methods. Also CXX-Qt adds wrapper code to around your invokables to automatically perform any conversion between the [C++ and Rust types](../concepts/types.md).
```

Comment:
`wrapper code to around` => `wrapper code around`

Reply:
Fixed